facebook/__init__.py

```diff
@@ -282,13 +282,23 @@ def fql(self, query):
         """
         return self.request(self.version + "/" + "fql", {"q": query})
 
-    def get_app_access_token(self, app_id, app_secret):
-        """Get the application's access token as a string."""
-        args = {'grant_type': 'client_credentials',
-                'client_id': app_id,
-                'client_secret': app_secret}
+    def get_app_access_token(self, app_id, app_secret, offline=False):
+        """
+        Get the application's access token as a string.
+        If offline=True, use the concatenated app ID and secret
+        instead of making an API call.
+        <https://developers.facebook.com/docs/facebook-login/
+        access-tokens#apptokens>
+        """
+        if offline:
+            return "%s|%s" % (app_id, app_secret)
+        else:
```

Comment:
Would it make sense to eliminate this "else", since if `offline` is True, the method will immediately return?